### DISCUSSION TITLE
Icon P1 Nano - Color feedback in LCD display?

### DISCUSSION THREAD

**Discussion by joeyxsam** [helgoboss/helgobox#1355] [Q&A]
I know this is possible, just not quite seeing how to do it with realearn and the p1 nano.  I've got track volume feedback showing up as dB on the display of an iCON P1 Nano (Mackie Supported), but I don't know how to get the text to appear with the track's color.  

Currently using this on the mapping:

Source 
Category: MIDI
Type:  Display (feedback only)
Protocol:  Mackie (LCD)
Display:  All
Line: 2

Target 
Type:  Track: set volume
Track:  Selected

Textual feedback:  Text Expression > Color > target.track.color

Thanks for any help!

**Comment by helgoboss** [comment]
The Mackie LCD protocol doesn't support any colors. So it must be something Nano-specific. The built-in display type "Mackie (LCD)" is therefore not enough. It either needs a new specialized one for the Nano or you need to roll your own with the MIDI script source (hard for non-programmers).
